arangod/Aql/QueryRegistry.h

```diff
@@ -122,6 +122,10 @@ class QueryRegistry {
   /// @brief return the default TTL value
   TEST_VIRTUAL double defaultTTL() const { return _defaultTTL; }
 
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
+  bool queryIsRegistered(std::string const& dbName, QueryId id);
```

Comment:
It would be nice if we could make this method `const`.

Reply:
This would require to make the lock mutable.
I am up for this, but I did not want to modify the "production" path in unnecessary ways here.